src/import/load.js

```diff
@@ -123,6 +131,13 @@ function xhr(url, callback) {
   }
 
   request.open('GET', url, async);
+  var headers = getHeaders(load.headers, opt.headers);
```

Comment:
can you use dl.extend() for this?   Also, I think it should be done inside the if (request.setRequestHeader) - otherwise why generate a new object and than not use it